Trivy-action is failing with no obvious reason

Hello!

I noticed today that trivy action stopped working in my GHA.
The logs are showing:
```
4s
Run aquasecurity/trivy-action@0.34.1
Run aquasecurity/setup-trivy@e6c2c5e321ed9123bda567646e2f96565e34abe1
Run echo "dir=$HOME/.local/bin/trivy-bin" >> $GITHUB_OUTPUT
Run actions/cache@0400d5f644dc74513175e3cd8d07132dd4860809
Cache not found for input keys: trivy-binary-v0.69.1-Linux-X64
Run actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8
Syncing repository: aquasecurity/trivy
Getting Git version info
Temporarily overriding HOME='/runner/_work/_temp/f9c4f44e-d13d-41f9-89f2-e8999506e885' before making global git config changes
Adding repository directory to the temporary git global config as a safe directory
/usr/bin/git config --global --add safe.directory /runner/_work/xxx/xxx/trivy
Initializing the repository
Disabling automatic garbage collection
Setting up auth
Determining the default branch
Fetching the repository
Determining the checkout info
Setting up sparse checkout
Checking out the ref
/usr/bin/git log -1 --format=%H
95883253cd45085b997451a38ccf8bb199a4e625
Run echo "installing Trivy binary"
installing Trivy binary
aquasecurity/trivy info checking GitHub for tag 'v0.69.1'
aquasecurity/trivy info found version: 0.69.1 for v0.69.1/Linux/64bit
Error: Process completed with exit code 1.
Run rm -f trivy_envs.txt
```
I tried using different versions, but without any luck. It was working fine previous time I checked (around 3 weeks ago).
My config:
```
- name: Run Trivy vulnerability scanner
  uses: aquasecurity/trivy-action@0.34.1
  with:
    scan-type: 'fs'
    scan-ref: '.'
    format: 'sarif'
```

--- a/action.yaml
+++ b/action.yaml
@@ -98,7 +98,7 @@ inputs:
   version:
     description: 'Trivy version to use'
     required: false
-    default: 'v0.69.1'
+    default: 'v0.69.2'
   cache:
     description: 'Used to specify whether caching is needed. Set to false, if you would like to disable caching.'
     required: false

